src/network/networkd-network-gperf.gperf

```diff
@@ -110,6 +110,9 @@ Bridge.AllowPortToBeRoot,               config_parse_bool,
 Bridge.UnicastFlood,                    config_parse_bool,                              0,                             offsetof(Network, unicast_flood)
 BridgeFDB.MACAddress,                   config_parse_fdb_hwaddr,                        0,                             0
 BridgeFDB.VLANId,                       config_parse_fdb_vlan_id,                       0,                             0
+BridgeVLAN.PVID,                        config_parse_uint32,                            0,                             offsetof(Network, pvid)
```

Comment:
To make this easy for you I added #3478 now, which fixes up that vlan ID type confusion. It also adds dedicated parsers for vlan IDs. Please have a look!

It might make sense to move you vlan range parser over to the new c files i added there too. Also, making use of parse_vlanid() in your range parsers, of course, would be good.